intg-denonavr/setup_flow.py

```diff
@@ -226,6 +226,16 @@ async def handle_configuration_mode(msg: UserDataResponse) -> RequestUserInput |
                 },
                 "field": {"checkbox": {"value": True}},
             },
+            {
+                "id": "volume_step",
+                "label": {
+                    "en": "Volume step",
+                    "fr": "Pallier de volume",
+                },
+                "field": {
+                    "number": {"value": 0.5, "min": 0.1, "max": 10, "steps": 0.1, "decimals": 1, "unit": {"en": "dB"}}
```

Comment:
The float definition in `steps: 0.1` triggers an error in the core service and setup always fails.
Even though the API defines it as a number, the core service can only handle integers at the moment.
I'll fix it within the next week and suggest to change it here to `steps: 1`. The web-configurator still allows to set float numbers.